scripts/builtin/img_translate_linearized.dml

```diff
@@ -0,0 +1,73 @@
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+# This function has  the same functionality with img_translate but it handles multiple images at
+# the same time. Each row of the input and output matrix represents a linearized image/matrix
+# It translates the image and Optionally resizes the image (without scaling).
+# INPUT:
+# ----------------------------------------------------------------------------------------------
+# img_in_linearized     Input matrix/image (every row represents a linearized matrix/image)
+# offset_x              The distance to move the image in x direction
+# offset_y              The distance to move the image in y direction
+# out_w                 Width of the output image
+# out_h                 Height of the output image
+# fill_value            the background of the image
+# original_w            width of the original 2D images
+# original_h            height of the original 2D images
+# --------------------------------------------------------------------------------------------
+#
+# OUTPUT:
+# --------------------------------------------------------------------------------------------
+# img_out_linearized  Output matrix/image  (every row represents a linearized matrix/image)
+# --------------------------------------------------------------------------------------------
+
+m_img_translate_linearized = function(Matrix[Double] img_in_linearized, 
+Double offset_x, Double offset_y, Integer out_w, Integer out_h, Double fill_value,
+Integer original_w, Integer original_h) return (Matrix[Double] img_out_linearized) {
```

Comment:
indent here with 4 space, to clearly indicate the function definition 

the rest of the file is indented with 2.